### ISSUE TITLE
Screen Readers tab to the second displayed panel instead of the main content.

### ISSUE THREAD

**Issue by JakeLaCombe** [cerner/terra-framework#263]
# Bug Report

## Description
When using the Slide Panel, the first content that gets accessed when browsing with the arrow keys is the panel content instead of the main content. Most likely, it should be the main content that is accessed first.

**Comment by bjankord** [comment]
Resolved in https://github.com/cerner/terra-framework/pull/289
